commands/rungearoutake.py

```diff
@@ -0,0 +1,20 @@
+import wpilib
+from wpilib.command import Command
+import subsystems
+import robotmap
+
+class RunGearOutake(Command):
+
+    def __init__(self, position):
+        super().__init__('Gear Outake')
+        self.requires(subsystems.gearoutake)
+        self.position = 0.0
+
+    def execute(self):
+        subsystems.gearoutake.set(self.position)
+
+    def isFinished(self):
+        super().isFinished()
```

Comment:
This always returns `False`. That means that this command will only end of it is cancelled or interrupted. I would look into making this command only run for the time period it takes to back out so that the gear is fully on the peg. This is tricky. Otherwise, you can make the gear door manually controlled by the driver.